app/controllers/api/v1/search_results_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchResultsController < ApplicationController
+      before_action :authorize!
+
+      def index
+        word = search_params[:adwords_url_contains]
+        pagination, keywords = get_keywords_has_ads_top_urls_contains_word(word)
+
+        render json: KeywordSerializer.new(keywords, meta: meta_from_pagination(pagination))
+      end
+
+      private
+
+      def authorize!
+        authorize :keyword
+      end
+
+      def search_params
+        params.require(:filter).permit(:adwords_url_contains)
+      end
+
+      def get_keywords_has_ads_top_urls_contains_word(word)
```

Comment:
In rails controller, the verb can be omitted in the method name

```suggestion
      def keywords_has_ads_top_urls_contains_word(word)
```